### PR TITLE
AB#58752 - Report Links Navigate to Resource Report

### PR DESCRIPTION
**Pull request by csmith-he**
[AB#58752](https://hedev.visualstudio.com/76e71f30-b1ad-438a-a096-89ac98712f4e/_workitems/edit/58752)

The link to a related resource now navigates to a resource editor page if the user is looking at a resource in the editor view, or the resource report page if a user is looking at a resource in the report view.

Permissions are still respected.

Peer reviewer: @khodgkinson-he 
Tester: @gouthamrandhi 
Merge lead: @aj-he

### CODE REVIEW

**Review by khodgkinson-he — APPROVED** [review]
I see, because..

image

`if(urlPathname.includes("resource")){`

Yep, yep, Yeppers! Looks good to me! ^_^

File: arches_her/media/js/utils/report.js [hunk 1]
```diff
@@ -190,7 +190,14 @@ define([
             if(node) {
                 const resourceId = node?.resourceId || node?.instance_details?.[0]?.resourceId;
                 if(resourceId){
-                    return `${arches.urls.resource}/${resourceId}`;
+                    urlPathname = window.location.pathname
+                    if(urlPathname.includes("resource")){
+                        return `${arches.urls.resource}/${resourceId}`;
+                    }
+                    else{
+                        return `${arches.urls.resource_report}${resourceId}`;
```
Comment (aj-he) [inline arches_her/media/js/utils/report.js:198]: @csmith-he you are missing a forward slash after `arches.urls.resource_report` compared to `arches.urls.resource`

Comment (csmith-he) [reply arches_her/media/js/utils/report.js:198]: @aj-he - For some reason the forward slash is already included in arches.urls.resource_report - if I put the forward slash in then you get // in the resulting URL

**Review by gouthamrandhi — APPROVED** [review]
Tested successfully.

**Review by aj-he — APPROVED** [review]
Looks good

**Pull request merged**